packages/tools/client-debugger/client-debugger-view/src/components/data-object-views/SharedMapView.tsx

```diff
@@ -30,10 +32,36 @@ export function SharedMapView(props: SharedMapViewProps): React.ReactElement {
 	const { sharedMap, renderChild } = props;
 
 	const [entries, setEntries] = React.useState<[string, unknown][]>([...sharedMap.entries()]);
+	const [collapsed, setCollapsed] = React.useState<{ [key: string]: boolean }>(() => {
+		const collapsedState: { [key: string]: boolean } = {};
+		// eslint-disable-next-line unicorn/no-array-for-each
+		entries.forEach(([key]) => {
+			collapsedState[key] = true;
+		});
```

Comment:
I don't think there's a good reason to go against the linter here, a regular `for (const [key] of entries) { collapsedState[key] = true; }` should do the trick too.

Reply:
Nice catch! Thank you 😄